pynpoint/readwrite/fitsreading.py

```diff
@@ -290,36 +288,36 @@ def _extra_attributes(self,
 
     def run(self):
         """
-        Run method of the module. Looks for all FITS files in the input directory and reads them
-        using the internal function _read_single_file(). Note that previous database information
-        is overwritten if overwrite=True. The filenames are stored as attributes. Note that
-        PynPoint only supports processing of square images so, in case required, images should
-        be made square with e.g. :class:`CropImagesModule` or :class:`RemoveLinesModule`.
+        Run method of the module. Looks for all FITS files in the input directory and imports the
+        images into the database. Note that previous database information is overwritten if
+        ``overwrite=True``. The filenames are stored as attributes.
 
         Returns
         -------
         NoneType
             None
         """
 
-        
         files = []
 
         if isinstance(self.m_filenames, str):
             files = self._txt_file_list()
-            location = ''
-        elif isinstance(self.m_filenames, list):
+            location = self._m_config_port.get_attribute("WORKING_PLACE")
+
+        elif isinstance(self.m_filenames, (list, tuple)):
             files = self.m_filenames
-            location = ''
+            location = self._m_config_port.get_attribute("WORKING_PLACE")
```

Comment:
This way the `location` will become the working directory of the pipeline. 
I don't think that is a good idea, as the user should ideally not interact with the working directory at all.